error-budgets/SLOs-SLIs.md

```diff
@@ -0,0 +1,60 @@
+# Definitions of SLOs and SLIs for packit
+
+If you are not familiar with SLOs and SLIs, please check [README.md](./README.md) first.
+
+## How do others do it?
+
+The CKI team has a pretty comprehensive set of
+[SLOs](https://cki-project.org/docs/hacking/rfcs/cki-004-slo/#created-slos-based-on-the-feedback).
+Instead of taking the same route, let's start small and slowly add more and
+more objectives and indicators over time.
+
+Testing Farm team has [Service level
+expectations](https://docs.google.com/document/d/1iHK3v_tCK4w-F82ZvUEO3xHcNv78tkR31w3oF-zpi1Q/edit#heading=h.26gd46u8e15q)
+and a pretty slick
+[dashboard](http://grafana-latest-osci-infra.cloud.privileged.psi.redhat.com/d/VXwH27XMk/rhel-infrastructure-and-service-health?orgId=1).
+
+Cockpit also uses this framework and the team has [a nice dashboard as well](http://grafana-latest-osci-infra.cloud.privileged.psi.redhat.com/d/VgtdtiwGk/rhel-infrastructure-and-service-health?orgId=1).
+
+## Packit SLOs
```

Comment:
Can we have a link to the [Users expectations](https://github.com/packit/research/blob/main/error-budgets/users-expectations.md) somewhere here?